### ISSUE TITLE
Ability to see current deployments of all stacks

### ISSUE THREAD

**Issue by teu** [Shopify/shipit-engine#531]
Hi,

this is more a usability feature - as we have over 10 stacks to deploy, some continuously, some not, would be great to be able to see what is being deployed on the initial shipit screen, which I can't right now  (or perhaps it's not documented ^^)

**Comment by byroot** [comment]
Hum, never though of that. Not a bad idea. It would require a bit of design though which is always Shipit's bottleneck.

Also not sure if you noticed, and it's not exactly what you ask for, but on the initial screen the stacks that have a :ship: icon are the ones with undeployed commits.

**Comment by teu** [comment]
Yes, the icon is not obvious though (would't have figured it out myself), I could suggest an on hover explanation on the icon may add to usability.
